modules/migration-configuring-cors-4.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// migration/migrating_openshift_3_to_4/migrating-openshift-3-to-4.adoc
+[id='migration-configuring-cors-4_{context}']
+= Configuring cross-origin resource sharing on {product-title} 4 clusters
+
+If you are migrating to or from an {product-title} 4 cluster that does not have the CAM tool installed, you must configure configure cross-origin resource sharing so that the CAM tool can access the cluster's API server.
+
+.Procedure
+
+. Log in to the CAM tool cluster.
+. Obtain the CAM tool URL:
```

Comment:
I would change:
"Obtain the CAM tool URL:" for "Obtain the CAM tool CORS configuration line:"

I actually thought I was getting a broken REGEX line before realizing it wasn't suppose to return the route, but the exact line config you need with the special characters.